app/helpers/orchestration_stack_helper/textual_summary.rb

```diff
@@ -56,11 +56,12 @@ def textual_ems_cloud
   def textual_orchestration_template
     template = @record.orchestration_template
     return nil if template.nil?
+    return nil unless template.orderable
     label = ui_lookup(:table => "orchestration_template")
     h = {:label => label, :image => "orchestration_template", :value => template.name}
     if role_allows(:feature => "orchestration_templates_view")
       h[:title] = _("Show this Orchestration Template")
-      h[:link] = url_for(:controller => 'catalog', :action => 'ot_show', :id => template.id)
+      h[:link] = url_for(:action => 'show', :id => @orchestration_stack, :display => 'stack_orchestration_template')
```

Comment:
@mzazrivec this link should be based upon orderable type, if template is orderable we should point by default to Catalog explorer and for non-orderable templates we need to redirect user to new screen so they can either copy or can make template as orderable. @bzwei your thoughts?